components/dashboard/src/admin/Settings.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { useContext } from "react";
+import { InstallationAdminSettings } from "@gitpod/gitpod-protocol";
+import { AdminContext } from "../admin-context";
+import CheckBox from "../components/CheckBox";
+import { PageWithSubMenu } from "../components/PageWithSubMenu";
+import { getGitpodService } from "../service/service";
+import { adminMenu } from "./admin-menu";
+
+export default function Settings() {
+    const { adminSettings, setAdminSettings } = useContext(AdminContext);
+
+    const actuallySetTelemetryPrefs = async (value: InstallationAdminSettings) => {
+        await getGitpodService().server.adminUpdateSettings(value);
+        setAdminSettings(value);
+    }
+
+    return (
+        <div>
+            <PageWithSubMenu subMenu={adminMenu} title="Settings" subtitle="Configure settings for your Gitpod cluster.">
+                <CheckBox
```

Comment:
**suggestion**: What do you think of adding a section header for this setting as we do for all other settings across the product? Not sure if the code suggestion here would work as expected. 😇 

```suggestion
                <h3>Usage Statistics</h3>
                <CheckBox
```